hollow/src/main/java/com/netflix/hollow/api/codegen/objects/HollowObjectJavaGenerator.java

```diff
@@ -148,6 +152,17 @@ private void appendAccessors(StringBuilder classBuilder) {
         }
     }
 
+    private static String generateFieldAccessorDeprecatedApiAnnotation(DeprecatedApi deprecatedApiAnnotation) {
+        StringBuilder builder = new StringBuilder();
+        if (deprecatedApiAnnotation != null) {
+            builder.append("    /**\n")
```

Comment:
If we remove the default value from the annotation, then you could check if `deprecatedApiAnnotation.value()` and only append the JavaDoc when it was non-null. But always add the `@Deprecated` annotation to the generated accessor.

Reply:
That way the following is also valid usage:

```
@DeprecatedApi
private final long prePaddingTime;
```

Reply:
For that matter, we could also support the core Java `@Deprecated` annotation for those cases  where you don't want a JavaDoc explanation:

```
@Deprecated
private final long prePaddingTime;
```